services/movie_session.py

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime, movie_id: int, cinema_hall_id: int
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet[MovieSession]:
+    manager = MovieSession.objects
```

Comment:
It is better to not assign manager to a variable, but assign a queryset using `all()` as it is done in the `get_movies`. Please, apply this change